.github/workflows/changelog.yml

```diff
@@ -0,0 +1,61 @@
+name: Changelog
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+jobs:
+  validate:
+    name: Validate changelog
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        # Include all history and tags
+        with:
+          fetch-depth: 0
+
+      # Ensure a new reno release note was added in this PR.
+      # Use `reno new <slug>` to add a new note to `releasenotes/notes`,
+      #   or add `changelog/no-changelog` label if no release note is needed.
+      - name: Ensure release note added
+        # Only run this on pull requests
+        if: github.event_name == 'pull_request'
+        run: |
+          jq -e '.pull_request?.labels[]?.name | select(. == "changelog/no-changelog")' ${{github.event_path}} \
```

Comment:
This might be better to move to an external script so it would be easier to render error/warning messages.

e.g.
> Warning: Skipping enforcement since changelog/no-changelog label is present

> Error: Release note not found, run "reno new <slug>" to create new release note for this PR

Reply:
We could also write a @Mergifyio rules that do this pretty simply FWIW. :)

Reply:
hehe, probably better to keep in GitHub actions if we can (reduce tool fatigue), but not opposed.

Reply:
Sure, I don't mind. I think there are many other features we could leverage there, such as the enforcing a workflow, merge queue or backports. Both me and @Kyle-Verhoog spend too much time clicking on buttons in GitHub to merge stuff. :(

That's another convo anyway.

Reply:
Yeah, there is a bunch of time wasted there. Since it doesn't effect me day to day as much as everyone else, not opposed to you using it!